src/platforms/common/traceswo.h

```diff
@@ -29,6 +29,7 @@
 void traceswo_init(uint32_t baudrate, uint32_t swo_chan_bitmask);
 void traceswo_deinit(void);
 uint32_t traceswo_get_baudrate(void);
+void bmp_usart_set_baudrate(uint32_t usart, uint32_t baud_rate);
```

Comment:
Given you're suffixing this (good), please use `bmd` not `bmp` - it's not a function specific to `native`, so should not use `native`'s name shortening but rather then project's. This is a point of consistency that needs work elsewhere in the code base still with this being the intended direction.

Reply:
I was looking at https://github.com/blackmagic-debug/blackmagic/blob/11d804d1fc5a094a5cf82978be1184a680484d7f/src/platforms/common/stm32/gpio.h#L28 for `bmp_gpio_set()`. You're right per `s/bmp_/bmd_/g`, just like in `-Dbmd_bootloader=true` case. Accepting.